
From: Pavel Machek <pavel@ucw.cz>

I'd like new swsusp to default to powerdown mode (as it did before) before
it goes up.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/kernel/power/main.c |    2 --
 1 files changed, 2 deletions(-)

diff -puN kernel/power/main.c~swsusp-fix-default-powerdown-mode kernel/power/main.c
--- 25/kernel/power/main.c~swsusp-fix-default-powerdown-mode	Tue Aug 17 15:16:29 2004
+++ 25-akpm/kernel/power/main.c	Tue Aug 17 15:16:29 2004
@@ -33,8 +33,6 @@ void pm_set_ops(struct pm_ops * ops)
 {
 	down(&pm_sem);
 	pm_ops = ops;
-	if (ops->pm_disk_mode && ops->pm_disk_mode < PM_DISK_MAX)
-		pm_disk_mode = ops->pm_disk_mode;
 	up(&pm_sem);
 }
 
_
